tools/bakeddata-scripts/Cargo.toml

```diff
@@ -11,8 +11,6 @@ edition = "2021"
 [dependencies]
 icu = { workspace = true, features = ["experimental"] }
 icu_compactdecimal = { workspace = true } # not in meta crate
```

Comment:
nit: turns out that compactdecimal is actually in the meta crate, consider fixing this while you're here. Would be nice to only have the meta crate, as the cut off for when something deserves baked data.

Reply:
done